lib/Parse/DNS/Zone.pm

```diff
@@ -408,6 +413,42 @@ sub _load_zonefile {
 	return do { local $/; <$zonefh> }; # slurp
 }
 
+sub _parse_time_unit {
+	my $string = shift;
+
+	my $time_units_regex = qr/^
+			(?: (?=\d+(?: w|d|h|m|s))
+			(\d+w)? (\d+d)? (\d+h)? (\d+m)? (\d+s)? ) | (\d+)
+		$/xi;
+	my @parts = ($string =~ $time_units_regex);
+
+	my %multi_for_unit = (
+		s => 1,
+		m => 60,
+		h => 3600,
+		d => 86400,
+		w => 604800,
+	);
+
+	my $seconds = 0;
+
+	for my $part (@parts) {
+		next
+			unless $part;
```

Comment:
I prefer these postfix `unless/if/for/etc` to be on a single line, when the expression is simple. *Especially* like in this case.